### PR TITLE
Add Spanish translation of README.md

### LINKED ISSUE #3: Add Spanish translation
After #2 opened by @cyb3rko, I think would be nice to add a Spanish translation of the CC Licenses and the `README.md`.
We can wait until #2 is merged, but my idea so far is having one directory for each translation. Like:

```
de
 ├── LICENSE-CC-BY
 ├── LICENSE-CC-BY-SA
 └── README.md 
es
 ├── LICENSE-CC-BY
 ├── LICENSE-CC-BY-SA
 └── README.md 
LICENSE-CC-BY
LICENSE-CC-BY-SA
README.md 
```

We should check if GitHub automatically recognizes the license out of a translated `LICENSE` file. If not, we should add a note encouraging people to add both the English and the translated version of the license text.

### PR DESCRIPTION
**Pull request by frxncismor**
Fixes #3.

I finished converting the Spanish translations of the README.

### CODE REVIEW

**Comment by cyb3rko** [comment]
If you take a look at my German README, I only included a badge for the main language otherwise you would have to include every translation and update it if any language gets added. Maybe you should change this.

Furthermore you still have to add the Spanish badge to the main README (and I slightly changed my badge as well, so you should include it as well):
`[German](de/)`
`[Spanish](es/)`

We used a slightly different syntax to name the chapters in the README, we used "CC" instead of "Creative Commons".
And the second chapter is missing "Internacional".

The internal links in the Spanish README are not working, use this (if you change the chapter names to "CC"):
`Aqui le presentamos dos tipos de licencias CC:`
`* [Creative Commons Atribución/Reconocimiento 4.0 Internacional](#cc-atribuciónreconocimiento-40-internacional)`
``
`* [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0](#cc-atribuciónreconocimiento-CompartirIgual-40-internacional)`

Behind the links I added in my translation the english name in brackets.

Greetings!

File: es/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Licencias Creative Commons para proyectos de Github
+
+[![Main](https://img.shields.io/badge/main%20language-EN-blue)](/../../)
+[![Deutsch](https://img.shields.io/badge/translation-DE-red)](de/)
+
+En este repositorio podrá encontrar formas sencillas de aplicar Licencias Creative Commons
+en repositorios Github a través del lenguaje Markdown.
```
Comment (santisoler) [inline es/README.md]:
```suggestion
en repositorios de Github a través del lenguaje Markdown.
```

File: es/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+> **ADVERTENCIA:**
+> No debería usar licencias Creative Commons para software.
```
Comment (santisoler) [inline es/README.md]:
```suggestion
> No debes utilizar licencias Creative Commons para software.
```

File: es/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+> Usa licencias de software libre para ese tipo de repositorios, como GPL, BSD, etc.
+> Las licencias de Creative Commons son solo para trabajos intelectuales.
+
+La manera más facil de encontrar la licencia CC adecuada para su proyecto es en el
+[sitio web Creative Commons](https://creativecommons.org/choose/).
+
+Aqui le presentamos dos tipos de licencias CC:
+* [Creative Commons Atribución/Reconocimiento 4.0 Internacional](#cc-attribution-40-international)
```
Comment (santisoler) [inline es/README.md]: Creative Commons use a shorter version of the license name on [their instruction deeds](https://creativecommons.org/licenses/by/4.0/deed.es): _Creative Commons Atribución 4.0 Internacional_. So, we might want to do the same in order to keep things simpler.
Besides, the link won't work as it is, so I changed that too.
```suggestion
* [Creative Commons Atribución 4.0 Internacional](#cc-atribución-40-internacional)
```

File: es/README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+* [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0](#cc-attribution-sharealike-40-international)
```
Comment (santisoler) [inline es/README.md]:
```suggestion
* [Creative Commons Atribución-CompartirIgual 4.0](#cc-atribución-compartirigual-40-internacional)
```

File: es/README.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Se puede obtener más información sobre licencias y textos de licencias con formato de texto sin formato
+en https://choosealicense.com/.
+
+Si quiere descargar la versión vectorizada de las imagenes de Creative Commons
+y otras por favor visite https://creativecommons.org/about/downloads/.
+
+## Creative Commons Atribución/Reconocimiento 4.0 Internacional
```
Comment (santisoler) [inline es/README.md]:
```suggestion
## CC Atribución 4.0 Internacional
```

File: es/README.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+Para agregar una licencia CC BY a su proyecto, simplemente agregue lo siguiente a su
+`README.md`. También debe copiar el [archivo de texto](LICENSE-CC-BY) de la licencia correspondiente y 
+renombrarlo a `LICENSE`.
+
+```markdown
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+Este trabajo está licenciado bajo [Creative Commons Atribución/Reconocimiento 4.0 Internacional][cc-by].
```
Comment (santisoler) [inline es/README.md]: El [deed de la licencia](https://creativecommons.org/choose/?lang=es_AR) recomienda utilizar este texto en español:

```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```

File: es/README.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: https://creativecommons.org/licenses/by/4.0/deed.es
+[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
+[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
+
+
+## Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0
```
Comment (santisoler) [inline es/README.md]:
```suggestion
## CC Atribución-CompartirIgual 4.0

```

File: es/README.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+Para agregar una licencia CC BY-SA a su proyecto, simplemente agregue lo siguiente a su
+`README.md`. También debe copiar el [archivo de texto](LICENSE-CC-BY-SA) de la licencia correspondiente y 
+renombrarlo a `LICENSE`.
+
+```markdown
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+Este trabajo está licenciado bajo
+[Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0][cc-by-sa].
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].
```

File: es/README.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+[![CC BY-SA 4.0][cc-by-sa-image]][cc-by-sa]
+
+[cc-by-sa]: https://creativecommons.org/licenses/by-sa/4.0/deed.es
+[cc-by-sa-image]: https://licensebuttons.net/l/by-sa/4.0/88x31.png
+[cc-by-sa-shield]: https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg
+```
+
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+Este trabajo está licenciado bajo [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0][cc-by-sa].
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].
```

File: es/README.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+La manera más facil de encontrar la licencia CC adecuada para su proyecto es en el
+[sitio web Creative Commons](https://creativecommons.org/choose/).
+
+Aqui le presentamos dos tipos de licencias CC:
+* [Creative Commons Atribución/Reconocimiento 4.0 Internacional](#cc-attribution-40-international)
+
+* [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0](#cc-attribution-sharealike-40-international)
+
+Se puede obtener más información sobre licencias y textos de licencias con formato de texto sin formato
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Puedes obtener más información sobre licencias y textos de licencias con formato de texto plano
```

File: es/README.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+en https://choosealicense.com/.
+
+Si quiere descargar la versión vectorizada de las imagenes de Creative Commons
+y otras por favor visite https://creativecommons.org/about/downloads/.
+
+## Creative Commons Atribución/Reconocimiento 4.0 Internacional
+
+Para agregar una licencia CC BY a su proyecto, simplemente agregue lo siguiente a su
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Para agregar una licencia CC BY a tu proyecto, simplemente agrega lo siguiente a tu
```

File: es/README.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+`README.md`. También debe copiar el [archivo de texto](LICENSE-CC-BY) de la licencia correspondiente y
```
Comment (santisoler) [inline es/README.md]:
```suggestion
`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY) de la licencia correspondiente y 
```

File: es/README.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: https://creativecommons.org/licenses/by/4.0/deed.es
+[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
+[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
+
+
+## Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0
+Para agregar una licencia CC BY-SA a su proyecto, simplemente agregue lo siguiente a su
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Para agregar una licencia CC BY-SA a tu proyecto, simplemente agrega lo siguiente a tu
```

File: es/README.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+`README.md`. También debe copiar el [archivo de texto](LICENSE-CC-BY-SA) de la licencia correspondiente y
```
Comment (santisoler) [inline es/README.md]:
```suggestion
`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY-SA) de la licencia correspondiente y 
```

File: es/README.md [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+
+En este repositorio podrá encontrar formas sencillas de aplicar Licencias Creative Commons
+en repositorios Github a través del lenguaje Markdown.
+
+> **ADVERTENCIA:**
+> No debería usar licencias Creative Commons para software.
+> Usa licencias de software libre para ese tipo de repositorios, como GPL, BSD, etc.
+> Las licencias de Creative Commons son solo para trabajos intelectuales.
+
+La manera más facil de encontrar la licencia CC adecuada para su proyecto es en el
```
Comment (santisoler) [inline es/README.md]:
```suggestion
La manera más facil de encontrar la licencia CC adecuada para tu proyecto es en el
```

File: es/README.md [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+[sitio web Creative Commons](https://creativecommons.org/choose/).
+
+Aqui le presentamos dos tipos de licencias CC:
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Aquí te presentamos dos tipos de licencias CC:
```

File: es/README.md [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+* [Creative Commons Atribución/Reconocimiento 4.0 Internacional](#cc-attribution-40-international)
+
+* [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0](#cc-attribution-sharealike-40-international)
+
+Se puede obtener más información sobre licencias y textos de licencias con formato de texto sin formato
+en https://choosealicense.com/.
+
+Si quiere descargar la versión vectorizada de las imagenes de Creative Commons
+y otras por favor visite https://creativecommons.org/about/downloads/.
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Si quieres descargar la versión vectorizada de las imagenes de Creative Commons
y otras, por favor visita https://creativecommons.org/about/downloads/.
```

**Review by santisoler — CHANGES_REQUESTED** [review]
Thank you very much @frxncismor!

I leave some small modifications:
- Use a non-formal Spanish,
- Modify suggested text according to the deeds from Creative Commons.

Please, take a look at them when you have time.

**Comment by frxncismor** [comment]
Ok i read all of your comments guys and i fixed the errors, hope i did well this time. I'm sorry it's my first contribution to open source

**Comment by cyb3rko** [comment]
@frxncismor @santisoler One more question for all of us:
Do we wanna add the original name of the licenses in the translated READMEs or do you think that it's unnecessary?

**Comment by frxncismor** [comment]
I think it's unnecessary because that information is on the main language

**Comment by cyb3rko** [comment]
Ok, then I will rework the German translation and yours is perfect!

File: es/README.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+
+En este repositorio podrás encontrar formas sencillas de aplicar Licencias Creative Commons
+en repositorios de Github a través del lenguaje Markdown.
+
+> **ADVERTENCIA:**
+> No debes utilizar licencias Creative Commons para software.
+> Usa licencias de software libre para ese tipo de repositorios, como GPL, BSD, etc.
+> Las licencias de Creative Commons son solo para trabajos intelectuales.
+
+La manera más facil de encontrar la licencia CC adecuada para tu proyecto es en el
```
Comment (santisoler) [inline es/README.md]:
```suggestion
La manera más fácil de encontrar la licencia CC adecuada para tu proyecto es en el
```

File: es/README.md [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+[sitio web Creative Commons](https://creativecommons.org/choose/).
+
+Aqui te presentamos dos tipos de licencias CC:
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Aquí te presentamos dos tipos de licencias CC:
```

File: es/README.md [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+* [Creative Commons Atribución 4.0 Internacional](#cc-atribución-40-internacional)
+
+* [Creative Commons Atribución-CompartirIgual 4.0 Internacional](#cc-atribución-compartirigual-40-internacional)
+
+Puedes obtener más información sobre licencias y textos de licencias con formato de texto plano
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Puedes obtener más información sobre licencias y sus textos completos en texto plano
```

File: es/README.md [hunk 21]
```diff
@@ -0,0 +1,82 @@
...
+## CC Atribución 4.0 Internacional
+
+Para agregar una licencia CC BY a tu proyecto, simplemente agrega lo siguiente a tu
+`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY) de la licencia correspondiente y
+renombrarlo a `LICENSE`.
+
+```markdown
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+Este trabajo está licenciado bajo [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```

File: es/README.md [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: https://creativecommons.org/licenses/by/4.0/deed.es
+[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
+[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
+```
+
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+Este trabajo está licenciado bajo [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```

File: es/README.md [hunk 23]
```diff
@@ -0,0 +1,82 @@
...
+
+## CC Atribución-CompartirIgual 4.0 Internacional
+Para agregar una licencia CC BY-SA a tu proyecto, simplemente agrega lo siguiente a tu
+`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY-SA) de la licencia correspondiente y
+renombrarlo a `LICENSE`.
+
+```markdown
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+Este trabajo está licenciado bajo
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Esta obra está bajo una
```

File: es/README.md [hunk 24]
```diff
@@ -0,0 +1,82 @@
...
+[![CC BY-SA 4.0][cc-by-sa-image]][cc-by-sa]
+
+[cc-by-sa]: https://creativecommons.org/licenses/by-sa/4.0/deed.es
+[cc-by-sa-image]: https://licensebuttons.net/l/by-sa/4.0/88x31.png
+[cc-by-sa-shield]: https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg
+```
+
+Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]
+
+Este trabajo está licenciado bajo [ Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].
```
Comment (santisoler) [inline es/README.md]:
```suggestion
Esta obra está bajo una
[Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].
```

**Review by santisoler — CHANGES_REQUESTED** [review]
Thanks @frxncismor for the modifications!

I leave some small changes, like missing tildes and an exact copy of the text proposed by Creative Commons. I think after you commit these changes we might be ready to merge!

**Comment by frxncismor** [comment]
I think it's done, please review it and thank you for give me the opportunity to contribute

**Review by santisoler — APPROVED** [review]

**Comment by santisoler** [comment]
Thanks a lot of tackling this @frxncismor!
I'll merge it right away.

I've just seen that you open this PR from the `master` branch of your fork.
As I told @cyb3rko on #2, your `master` will diverge from mine after I merge this. The easier way to solve it is to just delete your fork and refork it again if you want. But they way to prevent this is to create a branch where all your new commits will leave and then open the PR from that branch.
For example, @cyb3rko opened #5 from them `spanish` branch.
The good notice is that it's hard to break things in git and GitHub that aren't easily solved, so no big deal.

Thanks again for the contribution!

**Pull request merged**